pkg/envoy/api/http_connection_manager_test.go

```diff
@@ -110,3 +111,40 @@ func TestNewRouteConfig(t *testing.T) {
 
 	assert.DeepEqual(t, got, want, protocmp.Transform())
 }
+
+func TestNewHTTPConnectionManagerWithoutTrustedHopsDisabled(t *testing.T) {
```

Comment:
```suggestion
func TestNewHTTPConnectionManagerWithTrustedHops(t *testing.T) {
```
Your test name is also fine so please feel free to apply the suggestion or not by your preference.

Reply:
Nice spot, at first I wanted to create different test and then moved to a single loop test 😁